### PR TITLE
Add basic Api routes and logic

### PR DESCRIPTION
**Pull request by AlexisRoe**
added api routes on server and add the acquired logic in client (with mockups)

- [x] unix-timestamp tools
- [x] add env setup, MongoDriver
- [x] api routes: post /api/user for authentication

implementation of user authentication, send to api, checked with mongoDB and included error handling

- [ ] api routes: post /api/day for daily overview
- [ ] api routes: post /api/eventbyID

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -40,64 +40,64 @@
...
-                d="m 301,219.5 c -4.838,1.575 -9.523,3.583 -14,6 a 71.9,71.9 0 0 0 -11.75,8.5 l -16,9.75 c 0,0 -11.812,6.313 -23.5,8 a 89.74,89.74 0 0 1 -23,0"
-              />
-            </g>
             <path
-              id="path883"
-              stroke-width="5"
-              d="m 225,128.43 c 0,0 -10,13.223 -19.43,16.286 -9.43,3.063 -15.57,5 -19.714,4.143 -4.144,-0.857 -20.106,-26.36 -20.106,-26.36 0,0 9.2,13.48 8.25,24.75 -0.584,6.953 -6.7,13.1 -10.5,14.25 -4.513,1.357 -13.028,1.462 -20.5,-2.75 -7.84,-4.418 -14.582,-13.407 -17.75,-15.75 -6,-4.437 -10.964,-9.143 -23.75,-4.25 l -10,14 a 22.26,22.26 0 0 0 8,2.75 52.67,52.67 0 0 0 11,0 50.26,50.26 0 0 1 10.25,6 c 4.938,3.813 12.5,14.25 12.5,14.25"
+              id="path887"
+              d="m 92.143,153.57 c 0,0 -14.714,2.75 -18.286,34.57 0,0 3.536,-21.938 4.43,-34.57 C 79.181,140.938 76,138.714 76,138.714 c 0,0 1.146,-4.607 7.1,0.857 5.954,5.464 15.145,2.732 15.145,2.732"
+              stroke-linejoin="round"
```
Comment (ncbecker) [inline client/public/index.html:96]: You could optimize the svg with e.g. [Imageoptim](https://imageoptim.com) to save a few bytes and remove unnecessary code 😉

**Review by ncbecker — COMMENTED** [review]
Lgtm 👍

File: client/public/index.html (hunk 1 above)
Comment (AlexisRoe) [reply client/public/index.html:96]: 🎉 thanks ... thats a cool tool, didnt work with svg 😉

File: client/src/Pages/Login.js [hunk 2]
```diff
@@ -27,15 +27,14 @@ function Login() {
     try {
       const hashedPassword = hash(password);
       const response = await loginUser(dispatch, { id, hashedPassword });
-      console.log(response);
+      if (response) {
```
Comment (phlgr) [inline client/src/Pages/Login.js:30]:
```suggestion
      if (response.ok) {
```

File: server.js [hunk 3]
```diff
@@ -1,24 +1,85 @@
...
+    }
+
+    if (result.password !== request.query.pwd) {
+      response.status(401).send("Password incorrect");
+      return;
+    }
+
+    const auth_response = {
+      auth_token:
+        "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjoxfQ.5b0YUvXu9IFCI4kqzNAfrnuA2lSMp8XtezIZTfQYH4k",
```
Comment (phlgr) [inline server.js:52]: Why is this here?

File: client/src/Pages/Login.js (hunk 2 above)
Comment (AlexisRoe) [reply client/src/Pages/Login.js:30]: yeah right thanks, i forgot about that, thanks

File: server.js (hunk 3 above)
Comment (AlexisRoe) [reply server.js:52]: its just a placeholder, i will replace it later
